Backend/BlockChangesService.cs

```diff
@@ -11,12 +11,11 @@ namespace FRITeam.Swapify.Backend
 
     public class BlockChangesService : IBlockChangesService
     {
-        private readonly IMongoDatabase _database;
-        private IMongoCollection<BlockChangeRequest> _blockChangesCollection => _database.GetCollection<BlockChangeRequest>(nameof(BlockChangeRequest));
+        private readonly IMongoCollection<BlockChangeRequest> _blockChangesCollection;
 
         public BlockChangesService(IMongoDatabase database)
         {
-            _database = database;
+            _blockChangesCollection = database.GetCollection<BlockChangeRequest>(nameof(BlockChangeRequest));
         }
 
         public async Task AddAsync(BlockChangeRequest entityToAdd)
```

Comment:
Mongo should be able to generate Ids automatically...

Reply:
@ikim23 in whole app we assign ids manually